documentation/examples/remote_storage/remote_storage_adapter/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/blang/semver"
```

Comment:
I picked that library as its already vendored: https://github.com/prometheus/prometheus/tree/master/vendor/github.com/blang/semver